build/common.props

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup>
+        <!-- other options -->
+        <NeutralLanguage>en-US</NeutralLanguage>
+        <Company>Microsoft Corporation.</Company>
+        <Product>Microsoft .NET</Product>
+
+        <!-- pack options -->
+        <VersionPrefix>1.0.0</VersionPrefix>
+        <BuildNumber Condition="'$(BuildNumber)'==''">$([System.DateTime]::Now.Ticks)</BuildNumber>
+        <VersionSuffix>preview4-$(BuildNumber)</VersionSuffix>
+
+        <PackageId>$(MSBuildProjectName)</PackageId>
+        <Authors>Microsoft</Authors>
+        <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+        <!--<RepositoryUrl>https://github.com/aspnet/Scaffolding</RepositoryUrl>
```

Comment:
I assume commented out b/c of issues with NuGet verifier. Should be fixed with https://github.com/aspnet/BuildTools/pull/124